ush/ioda/bufr2ioda/bufr2ioda_tesac_mammals_profiles.py

```diff
@@ -0,0 +1,313 @@
+
+    # Datetime
+    obsspace.create_var('MetaData/dateTime', dtype=dateTime.dtype, fillval=dateTime.fill_value) \
+        .write_attr('units', 'seconds since 1970-01-01T00:00:00Z') \
+        .write_attr('long_name', 'Datetime') \
+        .write_data(dateTime)
+
+    # rcptDatetime
+    obsspace.create_var('MetaData/rcptdateTime', dtype=dateTime.dtype, fillval=dateTime.fill_value) \
+        .write_attr('units', 'seconds since 1970-01-01T00:00:00Z') \
+        .write_attr('long_name', 'receipt Datetime') \
+        .write_data(rcptdateTime)
+
+    # Longitude
+    obsspace.create_var('MetaData/longitude', dtype=lon.dtype, fillval=lon.fill_value) \
+        .write_attr('units', 'degrees_east') \
+        .write_attr('valid_range', np.array([-180, 180], dtype=np.float32)) \
+        .write_attr('long_name', 'Longitude') \
+        .write_data(lon)
+
+    # Latitude
+    obsspace.create_var('MetaData/latitude', dtype=lat.dtype, fillval=lat.fill_value) \
+        .write_attr('units', 'degrees_north') \
+        .write_attr('valid_range', np.array([-90, 90], dtype=np.float32)) \
+        .write_attr('long_name', 'Latitude') \
+        .write_data(lat)
+
+    # Station Identification
+    obsspace.create_var('MetaData/stationID', dtype=stationID.dtype, fillval=stationID.fill_value) \
+        .write_attr('long_name', 'Station Identification') \
+        .write_data(stationID)
+
+    # Depth
+    obsspace.create_var('MetaData/depth', dtype=depth.dtype, fillval=depth.fill_value) \
+        .write_attr('units', 'm') \
+        .write_attr('long_name', 'Water depth') \
+        .write_data(depth)
+
+    # Sequence Number
+    obsspace.create_var('MetaData/sequenceNumber', dtype=PreQC.dtype, fillval=PreQC.fill_value) \
+        .write_attr('long_name', 'Sequence Number') \
+        .write_data(seqNum)
+
+    # PreQC
+    obsspace.create_var('PreQC/waterTemperature', dtype=PreQC.dtype, fillval=PreQC.fill_value) \
+        .write_attr('long_name', 'PreQC') \
+        .write_data(PreQC)
+
+    obsspace.create_var('PreQC/salinity', dtype=PreQC.dtype, fillval=PreQC.fill_value) \
+        .write_attr('long_name', 'PreQC') \
+        .write_data(PreQC)
+
+    # ObsError
+    obsspace.create_var('ObsError/waterTemperature', dtype=ObsError_temp.dtype, fillval=ObsError_temp.fill_value) \
+        .write_attr('units', 'degC') \
+        .write_attr('long_name', 'ObsError') \
+        .write_data(ObsError_temp)
+
+    obsspace.create_var('ObsError/salinity', dtype=ObsError_saln.dtype, fillval=ObsError_saln.fill_value) \
+        .write_attr('units', 'psu') \
+        .write_attr('long_name', 'ObsError') \
+        .write_data(ObsError_saln)
+
+    # ObsValue
+    obsspace.create_var('ObsValue/waterTemperature', dtype=temp.dtype, fillval=temp.fill_value) \
+        .write_attr('units', 'degC') \
+        .write_attr('valid_range', np.array([-10.0, 50.0], dtype=np.float32)) \
+        .write_attr('long_name', 'Temperature below surface') \
```

Comment:
Same as before. You might want to keep these make these "below surface" be "water Temperature" unless it's guaranteed that all of the measurements have depth.

Reply:
I'll keep this as the main comment to stop this from getting messy, but keep this in mind for the other files too.